spec/presenters/pull_requests_by_team_spec.rb

```diff
@@ -141,6 +141,45 @@
           }
         ])
       end
+
+      context 'and there is an equal number of PRs for multiple applications' do
+        it 'orders the applications by name' do
+          team = Domain::Team.new(team_name: '#email', applications: ['signon', 'email-alert-api'])
+          pull_requests = [
+            {
+              application_name: 'signon',
+              title: 'Bump gds-api-adapters from 1.2.3 to 4.5.6',
+              url: 'https://www.github.com/alphagov/signon/pull/456',
+              opened_at: Date.parse('2018-01-01 08:00:00')
+            }, {
+              application_name: 'email-alert-api',
+              title: 'Bump gds-api-adapters from 1.2.3 to 4.5.6',
+              url: 'https://www.github.com/alphagov/email-alert-api/pull/456',
+              opened_at: Date.parse('2018-01-01 08:00:00')
+            }
+          ]
+
+          expect(described_class.new.execute(
+                   teams: [team],
+                   ungrouped_pull_requests: pull_requests
+          )).to eq([
```

Comment:
Tests look good, you may find that asserting the full response becomes a pain if you change the structure here.  Could get around that by only inspecting the order ` application_name ` appears.

Reply:
Agreed, but would still need to dig into the structure to assert on those specific values.
Going to leave this for now but valid point.